projects/driver_controls/src/can_output.c

```diff
@@ -0,0 +1,22 @@
+#include "can_output.h"
+#include "event_arbiter.h"
+#include "input_event.h"
+#include "log.h"
+
+void can_output_transmit(uint8_t device_id, uint8_t device_state, uint16_t device_data) {
+  union EventData {
```

Comment:
What's the point of this now? We have 64 bits to work with.

Reply:
Actually, that is a good point. I'll make a change to the function